### DISCUSSION TITLE
Host support

### DISCUSSION THREAD

**Discussion by ghost** [hathach/tinyusb#806] [Q&A] [answered]
Is there a plan or roadmap for stable support for Host mode operation? My use case is specifically around interfacing with CDC-ECM class devices, not sure if there is a plan there as well?

Thanks in advance!

**Comment by hathach** [comment, marked as answer]
There is no plan or roadmap available, there is no ETA since all the works is done in my spare time. But expanding host stack is on my todo list. Just keep watching.
